Assets/Scripts/Object/AbstractObject.cs

```diff
@@ -4,6 +4,7 @@
 namespace Object {
     public abstract class AbstractObject : MonoBehaviour {
         private const float defaultRotationSpeed = 42F;
+        private const double BH_radius = 4;
```

Comment:
![good](https://img.shields.io/badge/review-good-success.svg) 
nice to cut to constant!